### PR TITLE
update name form documentation

### PR DESCRIPTION
**Pull request by carodew**

### CODE REVIEW

File: _components/form-templates.md [hunk 1]
```diff
@@ -52,18 +52,27 @@ lead: Patterns for some of the most commonly used forms on government websites.
     Documentation
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
-    <h4 class="usa-heading">Usability</h4>
-    <ul class="usa-content-list">
-      <li>Leave the title and suffix fields as text boxes (instead of offering drop downs.) There are many possible titles and suffixes; text boxes accommodate them all.</li>
-      <li>Do not restrict the types of characters users can enter in any of these fields. Names can include characters outside the standard Roman alphabet.</li>
-    </ul>
-    <h4 class="usa-heading">Accessibility</h4>
+      <h4 class="usa-heading">Accessibility</h4>
```
Comment (maya) [inline _components/form-templates.md]: Indentation seems off here.

Comment (carodew) [inline _components/form-templates.md]: fixed in 6a6b5bf

File: _components/form-templates.md [hunk 2]
```diff
@@ -64,6 +59,20 @@ lead: Patterns for some of the most commonly used forms on government websites.
       <li><a href="{{ site.baseurl }}/form-controls/">the accessibility guidelines for form controls</a>.</li>
     </ul>
     <p>There are no other specific accessibility guidelines for this form template.</p>
+    <h4 class="usa-heading">Usability</h4>
+    <h5>When to use</h5>
+    <ul class="usa-content-list">
+      <li>When you need to collect users’ full names and store the parts separately in a database.</li>
+      </ul>
```
Comment (maya) [inline _components/form-templates.md]: This is should aline with the first `<ul>`

Comment (carodew) [inline _components/form-templates.md]: fixed in 2765104

**Pull request merged**